projects/1.18/assets/kitteh6660s-morecraft/morecraft/lang/zh_cn.json

```
@@ -0,0 +1,336 @@
+"item.morecraft.raw_spider": "蜘蛛",
+"item.morecraft.cooked_spider": "煎蜘蛛",
+"item.morecraft.raw_squid": "生鱿鱼",
+"item.morecraft.cooked_squid": "熟鱿鱼",
+"item.morecraft.raw_guardian_meat": "生守卫者肉",
+"item.morecraft.cooked_guardian_meat": "熟守卫者肉",
+"item.morecraft.raw_dragon_meat": "生龙肉",
+"item.morecraft.cooked_dragon_meat": "熟龙肉",
+"item.morecraft.cooked_egg": "煎蛋",
+"item.morecraft.nether_apple": "下界苹果",
+"item.morecraft.cake_slice": "蛋糕块",
+"item.morecraft.apple_pie": "苹果派",
+"item.morecraft.nether_apple_pie": "下界苹果派",
+"item.morecraft.sweetberry_pie": "甜浆果派",
+
+"item.morecraft.beef_stew": "牛肉煲",
+"item.morecraft.pork_stew": "猪肉煲",
+"item.morecraft.mutton_stew": "羊肉煲",
+"item.morecraft.chicken_stew": "鸡汤",
+"item.morecraft.chevon_stew": "山羊煲",
+"item.morecraft.spider_stew": "蜘蛛煲",
+"item.morecraft.fish_stew": "鱼杂烩",
+"item.morecraft.guardian_stew": "守卫者煲",
+"item.morecraft.dragon_stew": "龙煲",
+"item.morecraft.nethershroom_stew": "下界真菌炖菜",
+
+"item.morecraft.cod_sushi": "鳕鱼寿司",
+"item.morecraft.salmon_sushi": "鲑鱼寿司",
+"item.morecraft.squid_sushi": "鱿鱼寿司",
+
+"item.morecraft.patina": "铜锈",
+"item.morecraft.copper_nugget": "铜粒",
+"item.morecraft.ruby": "红宝石",
+"item.morecraft.wither_bone": "凋灵之骨",
+"item.morecraft.wither_bone_meal": "凋灵之骨粉",
+"item.morecraft.guardian_scales": "守卫者鳞片",
+"item.morecraft.enderdragon_scales": "末影龙鳞",
+
+"item.morecraft.netherwood_boat": "下界木船",
+"item.morecraft.warped_boat": "诡异木船",
+"item.morecraft.crimson_boat": "绯红木船",
+"item.morecraft.oak_crafting_table_minecart": "橡木工作台矿车",
+"item.morecraft.spruce_crafting_table_minecart": "云杉木工作台矿车",
+"item.morecraft.birch_crafting_table_minecart": "白桦木工作台矿车",
+"item.morecraft.jungle_crafting_table_minecart": "丛林木工作台矿车",
+"item.morecraft.acacia_crafting_table_minecart": "金合欢木工作台矿车",
+"item.morecraft.dark_oak_crafting_table_minecart": "深色橡木工作台矿车",
+"item.morecraft.crimson_crafting_table_minecart": "绯红木工作台矿车",
+"item.morecraft.warped_crafting_table_minecart": "诡异木工作台矿车",
+"item.morecraft.netherwood_crafting_table_minecart": "下界木工作台矿车",
+
+"block.morecraft.netherbrick_door": "下界砖门",
+"block.morecraft.netherwood_door": "下界木门",
+"block.morecraft.netherite_door": "下界合金门",
+"block.morecraft.glass_door": "玻璃门",
+"block.morecraft.soul_glass_door": "离魂玻璃门",
+"block.morecraft.bone_door": "骨门",
+"block.morecraft.netherbrick_trapdoor": "下界砖活板门",
+"block.morecraft.netherwood_trapdoor": "下界木活板门",
+"block.morecraft.netherite_trapdoor": "下界合金活板门",
+"block.morecraft.glass_trapdoor": "玻璃活板门",
+"block.morecraft.soul_glass_trapdoor": "离魂玻璃活板门",
+"block.morecraft.bone_trapdoor": "骨活板门",
+"block.morecraft.netherbrick_fence_gate": "下界砖栅栏门",
+"block.morecraft.netherwood_fence_gate": "下界木栅栏门",
+
+"block.morecraft.tall_netherbrick_door": "高大的下界砖门",
+"block.morecraft.tall_netherwood_door": "高大的下界木门",
+"block.morecraft.tall_netherite_door": "高大的下界合金门",
+"block.morecraft.tall_glass_door": "高大的玻璃门",
+"block.morecraft.tall_soul_glass_door": "高大的离魂玻璃门",
+"block.morecraft.tall_bone_door": "高大的骨门",

```

```suggestion
"block.morecraft.tall_netherbrick_door": "高下界砖门",
"block.morecraft.tall_netherwood_door": "高下界木门",
"block.morecraft.tall_netherite_door": "高下界合金门",
"block.morecraft.tall_glass_door": "高玻璃门",
"block.morecraft.tall_soul_glass_door": "高灵魂玻璃门",
"block.morecraft.tall_bone_door": "高骨门",
```
需查看是不是三格高的门，同作者有个dramaticdoors，疑似对此模组的联动